src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -86,14 +86,13 @@ def get_mdtest_job_manager_command(self, manager):
             mpio_util = MpioUtils()
             if mpio_util.mpich_installed(self.hostlist_clients) is False:
                 self.fail("Exiting Test: Mpich not installed")
-            self.job_manager = Mpirun(self.mdtest_cmd, mpitype="mpich")
+            self.job_manager = Mpirun(self.mdtest_cmd, self.subprocess, mpitype="mpich")
         else:
-            self.job_manager = Orterun(self.mdtest_cmd)
+            self.job_manager = Orterun(self.mdtest_cmd, self.subprocess)
 
         return self.job_manager
 
-    def run_mdtest(self, manager, processes, display_space=True, pool=None,
-                   out_queue=None):
+    def run_mdtest(self, manager, processes, display_space=True, pool=None, out_queue=None):
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.